core/chains/evm/headtracker/heads.go

```diff
@@ -6,28 +6,18 @@ import (
 
 	"github.com/ethereum/go-ethereum/common"
 
+	htrktypes "github.com/smartcontractkit/chainlink/v2/common/headtracker/types"
 	evmtypes "github.com/smartcontractkit/chainlink/v2/core/chains/evm/types"
 )
 
-// Heads is a collection of heads. All methods are thread-safe.
```

Comment:
Just noting that earlier, this interface was internal to the HeadTracker.
Now that you are exporting it to external callers, not everything here makes sense to be exported.
Only the ones that are needed by external callers.

Reply:
What do you mean? This interface is exported from a non-internal directory in both cases. What changed?

Reply:
I might be missing something here,  but it seemed to me like the initial `Heads` interface from `core/chains/evm/headtracker/heads.go` is exported and it is not internal to the `headtracker` package. 

Am I missing something here, or why would the `Heads Interface be internal`?